### ISSUE TITLE
Feature Request: Allow lapis.conf to be configured via Heroku's config vars

### ISSUE THREAD

**Issue by Shugabuga** [kupiakos/LapisMirror#36]
This can make the user's bots more secure if they are running off of a Heroku instance connected with GitHub (as I am doing like a complete idiot). It's briefly explained [here](https://gist.github.com/hzsweers/8595628), but a tl;dr is that you need to add vars with [in Terminal]...

```
$ heroku config:set REDDIT_USERNAME=<username>
```

...and to retrieve them with [in Python]...

```
login_info = [os.environ['REDDIT_USERNAME']
```

Idk if you want to save this for 2.0 or something, but I think that it would be a welcome addition to LapisMirror for getting everything set up easily and (more) securely.

**Comment by kupiakos** [comment]
Yeah that's actually a feature that's in the works in Lapis 2.0 (already have some of the code for that working). May have to scale back 2.0 a little bit to add these nice little features.

What I do right now is have a separate branch that's tracking the Heroku repo that has a fully configured `lapis.conf` with API keys and such, and merge changes from `master` into `herokudeploy` then `git push`.

**Comment by Shugabuga** [comment]
Yeah, I should be doing something like that by making a separate repo for Heroku to track. Maybe I'll fix that when I am free next.
